api/proto/teleport/statichostuser/v1/statichostuser.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+package teleport.statichostuser.v1;
+
+import "teleport/header/v1/metadata.proto";
+import "teleport/legacy/types/wrappers/wrappers.proto";
+
+option go_package = "github.com/gravitational/teleport/api/gen/proto/go/teleport/statichostuser/v1;statichostuserv1";
+
+// StaticHostUser is a resource that represents host users that should be
+// created on matching nodes.
+message StaticHostUser {
+  // kind is a resource kind.
+  string kind = 1;
+  // sub_kind is an optional resource sub kind, used in some resources.
+  string sub_kind = 2;
+  // version is the resource version. It must be specified.
+  // Supported values are: `v1`.
+  string version = 3;
+  // metadata is resource metadata.
+  teleport.header.v1.Metadata metadata = 4;
+  // spec is the static host user spec.
+  StaticHostUserSpec spec = 5;
+}
+
+// StaticHostUserSpec is the static host user spec.
+message StaticHostUserSpec {
+  // login is the login to create on the node.
+  string login = 1;
+  // groups is a list of additional groups to add the user to.
+  repeated string groups = 2;
+  // sudoers is a list of sudoer entries to add.
+  repeated string sudoers = 3;
+  // uid is the new user's uid.
+  string uid = 4;
+  // gid is the new user's gid.
+  string gid = 5;
+  // node_labels is a map of node labels (used to dynamically grant access to
+  // nodes).
```

Comment:
Here and below, update the comment since the labels here are used to select nodes, not to grant access to nodes.

```suggestion
  // node_labels is a map of node labels that will create a user
  // for this resource
```